unified-installer/astra-unified-installer.sh

```diff
@@ -2162,7 +2307,7 @@ step_add_labels_to_kustomization() {
     logheader $__DEBUG "Adding labels to kustomization and crs file"
 
     local -r content="labels:${__NEWLINE}- pairs:${__NEWLINE}${processed_labels}"
-    insert_into_file_after_pattern "${kustomization_file}" "kind:" "${content}"
+    insert_into_file_after_pattern "${kustomization_file}" "kind: Kustomization" "${content}"
```

Comment:
Some in-line kustomization patches have the `kind:` field, so I'm making this pattern more specific to avoid inserting the content in the wrong place (i.e. in the middle of a patch definition).